src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -44,13 +46,28 @@ function ExamSettings(props) {
   }
 
   function onButtonClick() {
+    if (proctoringProvider === 'proctortrack' && !EmailValidator.validate(proctortrackEscalationEmail)) {
+      setProctortrackEscalationEmailError('A valid escalation email must be provided if '
+                                            + 'Proctortrack is the selected provider.');
+    } else {
+      setProctortrackEscalationEmailError('');
     // TODO: implement POST
+    }
   }
 
+  // function isDisabledOption(provider) {
```

Comment:
Please review this commented out code.

Reply:
Noted, this isn't quite ready for review yet